docs/source/models/cause.rst

```diff
@@ -275,14 +275,68 @@ Birth Prevalence
+
+Incidence Rates
+^^^^^^^^^^^^^^^
+
+Generally, incidence is a measure of new cases of a given condition that occur in a given timeframe and population. The count value of new cases of the condition of interest will always be the numerator of incidence measures. The denominator of incidence measures is somewhat more complex and is critical to ensuring an accurate data source to inform cause model transition rates.
+
+Incidence rates that are estimated by the Global Burden of Disease study are measures of *cumulative incidence.* Cumulative incidence is a defined as the following: 
+
+.. todo: 
+
+	insert formatted equation for: new cases of condition / person-time of at risk population
+
+
+**Person-time:** person-time is a measure of the number of individuals multiplied by the amount of time they individually occupy the population of interest.
+
+	For example, if one individual is in our population of interest for two years, they contribute two person-years. If another individual is in our population of interest for 6 months, they contribute 0.5 person-years. Together, these individuals contribute a total of 2.5 person-years.
+
+**At-risk population:** the at-risk population is defined as the population that *does not* have the condition of interest; in other words, the susceptible population that is at risk of developing the condition. Notably, the number of individuals in this population will change overtime as the following events occur:
+
+	- Members of the at-risk population develop of the condition and are no longer susceptible
+	- Members of the at-risk population die and are no longer susceptible 
+	- Individuals are born or age into the at-risk population and become susceptible
+	- Individuals with the condition recover from the condition and re-enter the at-risk population as susceptible (in the case of conditions with remission)
+
+Cumulative incidence contrasts with other measures of incidence that may have different denominators. For instance, incidence rate may refer to new cases of a given condition that occur in a specified timeframe (e.g. 10 cases per year) or may refer to new cases per person-time of the general population rather than the at-risk population. Incidence measures other than cumulative with person-time of the at-risk population as the denominator are not appropriate for incidence data sources for cause model transition probabilities.
+
+Because the denominator for cumulative incidence is person-time in the at-risk population, it can represent the probability of a new case of the condition occuring in an individual without the condition in a given time frame. Therefore, it can be used to represent the probability that a simulant will transition from a susceptible to infected cause model state in a given timestep.
+
+	For instance, as reported by the GBD, the global incidence of injuries in 2017 was approximately 6,800 cases per 100,000 person-years, or 0.068 cases per person-year.
```

Comment:
Do the GBD incidence measures mean person-years for the total population as opposed to the at-risk population? It would be good if we clarified this somewhere.

Reply:
If so, then in the example of injuries below, we're approximating at-risk person-years by total person-years, which should be valid because the injury rate is low. Is that correct?

Reply:
This is addressed in most recent commit - thanks for the feedback!